docker/fuzzbench/Dockerfile

```diff
@@ -31,7 +31,10 @@ COPY benchmarks benchmarks
 COPY common common
 COPY database database
 COPY docker docker
+COPY experiment/build experiment/build
+COPY experiment/*.py experiment/
 COPY fuzzbench fuzzbench
 COPY fuzzers fuzzers
+COPY config.yaml config.yaml
```

Comment:
nit: can we make this more descriptive by calling it local-experiment-config.yaml?

Reply:
Done.

Reply:
Yes. Done.